Submodels/Parameterizations.py

```
@@ -0,0 +1,761 @@
+    def __init__(self) -> None:
+        """
+        Initialize the AirfoilParameterization class.
+        
+        This method sets up the initial state of the class.
+
+        Returns
+        -------
+        None
+        """
+        
+        return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)